### PR TITLE
assignment17

### LINKED ISSUE #457: [DOC]
**Provide a link to that doc page:**
https://github.com/fluid-cloudnative/fluid/blob/master/pkg/utils/charts.go
**What is the defect and your suggestions on improvement:**
类别： 方法注释规范化 位置： pkg/utils/charts.go 内容： 将方法func GetChartsDirectory上的注释规范化

### PR DESCRIPTION
**Pull request by hxycode**
### Ⅰ. Describe what this PR does
Normalize the annotation on SaveConfigMapToFile function.(pkg/utils/charts.go)

### Ⅱ. Does this pull request fix one issue?

fixes #457 

### Ⅴ. Special notes for reviews

### CODE REVIEW

File: pkg/utils/charts.go [hunk 1]
```diff
@@ -29,7 +29,10 @@ func PathExists(path string) bool {
 
 var chartFolder = ""
 
-// Get the directory of charts
+/**
+*
+* Get the directory of charts
+**/
```
Comment (xieydd) [inline pkg/utils/charts.go:35]: I think // is ok : )

**Review by xieydd — CHANGES_REQUESTED** [review]

File: pkg/utils/charts.go [hunk 2]
```diff
@@ -29,7 +29,10 @@ func PathExists(path string) bool {
 
 var chartFolder = ""
 
-// Get the directory of charts
+/**
+*
+* Get the directory of charts
```
Comment (TrafalgarZZZ) [inline pkg/utils/charts.go:34]: Please follow the godoc [convention](https://blog.golang.org/godoc) on comments. That is, using func name as the inital word. Thanks.

**Review by TrafalgarZZZ — CHANGES_REQUESTED** [review]

File: pkg/utils/charts.go (hunk 2 above)
Comment (hxycode) [reply pkg/utils/charts.go:34]: are you sure?other classmate did not use the func name.

Comment (hxycode) [reply pkg/utils/charts.go:34]: image

Comment (TrafalgarZZZ) [reply pkg/utils/charts.go:34]: Yeah, I'm sure.  You are suggested to do something like this:
```suggestion
// GetChartsDirectory gets the directory of the charts
```

**Review by cheyang — APPROVED** [review]

**Pull request merged**